index.html

```diff
@@ -1,210 +1,272 @@
 
-      <div class="vertical-line"></div>
+      <p class="work-p">
+        A daily selection of privately personalized reads; no accounts or
+        sign-ups required. has been the industry's standard dummy text ever
+        since the 1500s, when an unknown printer took a standard dummy text.
+      </p>
 
-      <p>Ruby</p>
-    </div>
+      <div class="vertical-lines flex">
+        <p>css</p>
 
-    <button type="button">See Project</button>
+        <div class="vertical-line"></div>
 
-    <div class="white-screen"></div>
+        <p>html</p>
 
-    <h2>Multi-Post Stories</h2>
+        <div class="vertical-line"></div>
 
-    <p class="work-p">
-      A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
-    </p>
+        <p>bootstrap</p>
 
-    <div class="vertical-lines flex">
-      <p>css</p>
+        <div class="vertical-line"></div>
 
-      <div class="vertical-line"></div>
+        <p>Ruby</p>
+      </div>
 
-      <p>html</p>
+      <button type="button">See Project</button>
+    </main>
 
-      <div class="vertical-line"></div>
+    <footer>
+      <section>
+        <h2>CONTACT US</h2>
 
-      <p>bootstrap</p>
+        <p class="footer-p">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+      </section>
 
-      <div class="vertical-line"></div>
+      <form class="flex" action="https://formspree.io/f/mayajwdo" method="POST">
 
-      <p>Ruby</p>
-    </div>
+        <fieldset>
+          <label for="name"></label>
+          <input type="text" name="name" id="name" placeholder="Text input" minlength="5" maxlength="30" required/>
+  
+          <label for="email"></label>
+          <input type="email" name="email" id="email" placeholder="Email address" required/>
 
-    <button type="button">See Project</button>
+          <textarea name="user-message" placeholder="Enter text here" minlength="3" maxlength="500" required></textarea>
+        </fieldset>
 
-  </main>
+        <input class="btn-center" type="submit" value="See Project">
+      </form>
 
-  
-</body>
-</html>
\ No newline at end of file
+      <hr>
+
+      <div class="icons-footer flex">
+        <div>
+          <a href="#"><img src="/icons/github.png" alt="github" /></a>
+        </div>
+
+        <div>
+          <img src="/icons/in.png" alt="linkedin" />
+        </div>
+
+        <div>
+          <img src="/icons/2.png" alt="Angelist" />
+        </div>
+
+        <div>
+          <img src="/icons/twitter.png" alt="twitter" />
+        </div>
+
+        <div>
+          <img src="/icons/M.png" alt="medium" />
+        </div>
+      </div>
+
+      <div class="link-email">
+        <a href="#">eri89--@hotmail.com</a>
+      </div>
```

Comment:
I suggest you wrap these social link list in a `ul` tag and each social link with `li` tag this is best practice